### PR TITLE
[Bug Fix] LPS-107347

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang, the pr is based on the [requirement](https://github.com/liferay-core-infra/liferay-portal/pull/127#issuecomment-1002557978).

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by yuhai** [comment]
ci:test

File: portal-impl/src/com/liferay/portal/setup/SetupWizardUtil.java [hunk 1]
```diff
@@ -193,7 +193,8 @@ public static void updateSetup(
 		_updateCompany(httpServletRequest, unicodeProperties);
 
 		_updateAdminUser(
-			httpServletRequest, httpServletResponse, unicodeProperties);
+			httpServletRequest, httpServletResponse, unicodeProperties,
+			databaseConfigured);
```
Comment (dantewang) [inline portal-impl/src/com/liferay/portal/setup/SetupWizardUtil.java:197]: @yuhai Here the variable name is pretty confusing. Actually `databaseConfigured == true` means the database is **NOT** changed by setup wizard. Could you add an SF commit before this commit to give it a more straightforward name, like "databaseConfigurationUnchanged" or something?

**Review by dantewang — CHANGES_REQUESTED** [review]
@yuhai Please make changes as requested. You can force push to this pull.

**Comment by dantewang** [comment]
ci:stop

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by yuhai** [comment]
ci:test

**Comment by yuhai** [comment]
@dantewang, I just force push new changes for the above requirement.

**Comment by dantewang** [comment]
ci:close

Please see https://github.com/yuhai/liferay-portal/pull/277#issuecomment-1003027194

**Comment by dantewang** [comment]
@zhangyan0701 Is ci-test broken?

**Comment by zhangyan0701** [comment]
@dantewang  There is a delay when Jenkins importing test report to Testray.

**Pull request closed without merging**